.gitignore

```diff
@@ -157,3 +157,6 @@ sftp-config.json
 
 .env
 dist/
+
+## graphql - generated code
```

Comment:
These files were generated somehow while any build step, so I added this line, to not accidentally adding them.